README.md

```diff
@@ -1,56 +1,61 @@
-
-> **Uwaga!** Możliwe, że w powyższym rozszerzeniu będziesz musiał użyć odpowiedniej skali obrazu wykorzystywanego jako podgląd. Jest to zależne od [ustawień systemowych](https://pliki.wiki/blog/jak-korzystac-ze-skalowania-ekranu-w-systemie-windows-10/). Zazwyczaj jednak domyślne ustwienia wtyczki dobrze się sprawdzają i poza jej instalacją nie musisz nic robić.
-
-Strona nie musi być wykonana zgodnie z pierwowzorem 1:1. Ważne, aby na pierwszy rzut oka było widać, że jest to ten sam projekt. 
-
-Pamiętaj, że przedstawiona grafika to element statyczny, a strona internetowa to element dynamiczny, który powinien dobrze wyglądać w różnych rozdzielczościach / na różnej wielkości monitorach. Część z nich może mieć szerokość ekranu 1280px, a inna 1440px itp. Na razie jednak skup się na odwzorowaniu zamieszczonego w zadaniu widoku. Innymi rozdzielczościami zajmiemy się w kolejnym module.
-
-## Propozycja podziału
-
-Poniżej zamieszczam propozycję podziału strony na poszczególne grupy:
-
-- wiersz (`width: 100%`) – kolor zielony
-- wyśrodkowanie (`max-width: 1150px`) – kolor niebieski
-- komórka (wiele elementów w jednej linii) – kolor pomarańczowy.
-
-Nie musisz się od tego stosować – to jedynie moja koncepcja, która ma Ci pomóc w utworzeniu odpowiedniej struktury.
-
-![](./assets/demo-info.png)
-
-&nbsp;
-
-> :warning: Jeśli nie posiadasz materiałów do tego zadania, to znajdziesz je na stronie [devmentor.pl](https://devmentor.pl/p/html-and-css-basics/).
-
-
-&nbsp;
-
-> ⭐ ***README** to coś więcej niż opis. Poprzez nie **pokazujesz swoje mocne strony** – swoją dokładność, sposób myślenia i podejście do rozwiązywania problemów. Niech Twoje README pokaże, że masz **świetne predyspozycje do rozwoju!***
-> 
-> 🎁 *Zacznij od razu. Skorzystaj z **[szablonu README i wskazówek](https://github.com/devmentor-pl/readme-template)**.* 
+# Czego się nauczyłem:
+
+- Zdobycie praktycznej wiedzy na temat struktury HTML i stylowania CSS.
+- Wykorzystanie różnych tagów HTML takich jak `<header>`, `<section>`, `<article>`, `<figure>`, `<figcaption>`, `<ul>`, `<li>`, `<aside>`, itp.
+- Implementacja podstawowych stylów CSS, takich jak kolorowanie tekstu, tło, układanie elementów, czy korzystanie z jednostek procentowych i pikselowych.
+
+# Jakich rozwiązań użyłem:
+
+- Poprawienie struktury HTML na podstawie uwag mentora, uwzględniając zalecenia dotyczące semantyki, takie jak zastępowanie niektórych `<div>` tagami bardziej adekwatnymi do znaczenia, np `<header>`, `<aside>`, `<article>`, `<figure>`.
+
+    - Przykład
+```
+<header>
+    <h1>Nagłówek</h1>
+</header>
+```
+- Zaimplementowanie poprawek stylów CSS, aby dostosować wygląd strony do oczekiwań mentora.
```

Comment:
Wygląda jakbym "ja" był tutaj najważniejszy, a nie o to chodzi. Może lepiej będzie napisać dopracowanie wyglądu zgodnie z wymaganiami projektu.